helm/k8s-dns-node-cache-app/values.yaml

```diff
@@ -9,7 +9,7 @@ serviceType: managed
 image:
   registry: quay.io
   name: giantswarm/k8s-dns-node-cache
-  tag: 1.21.4
+  tag: 1.22.13
```

Comment:
You might want to bump here as well https://github.com/giantswarm/k8s-dns-node-cache-app/blob/main/helm/k8s-dns-node-cache-app/Chart.yaml#L2

Reply:
Done